README.md

```diff
@@ -193,7 +193,7 @@ filters:
 
 provider:
   type: webSocket
-  timeout: 10000 #unit: ms
+  timeout: 10000 # unit: ms. This is optional value. The default is 10000 
```

Comment:
It might be good to explain more about `timeout`. 
What is the purpose of this? or How it works?